__docs__/wonder-blocks-dropdown/single-select-testing-snapshots.stories.tsx

```diff
@@ -72,6 +80,11 @@ export const StateSheetStory: Story = {
         );
     },
     parameters: {
-        pseudo: defaultPseudoStates,
+        pseudo: {
+            ...defaultPseudoStates,
+            // Using the focus selector instead so that the focus outline is shown
```

Comment:
Thanks for adding a comment here! What an odd scenario where `focus` is preferable for once. Is it because it's technically programmatic focus?

Reply:
It is so the focus behaviour looks more similar to other form fields! This PR comment shows the difference between using `focus` and `focus-visible`: https://github.com/Khan/wonder-blocks/pull/2873#discussion_r2566062354 

There is also a comment where we set the focus styles for context!

https://github.com/Khan/wonder-blocks/blob/a7993883ce697d28ddd5ef2112eadff67f4d359f/packages/wonder-blocks-dropdown/src/components/select-opener.tsx#L262-L267